indigo_app/templates/main.html

```diff
@@ -9,139 +9,135 @@
   </div>
 
   <div class="page-body">
-    {% block sidebar %}{% endblock %}
-
-    <div class="main-container-wrapper">
-      <main id="{% block view-id %}{% endblock %}" class="main-container">
-        {% block navbar %}
-          <nav class="navbar navbar-expand-md navbar-dark">
-            {% block navbar-brand %}
-            <a class="navbar-brand" href="/"><img src="{% static 'images/logo-inverse.png' %}" title="Indigo"></a>
-            {% endblock %}
-            {% block navbar-toggle %}
-            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar-collapse">
-              <span class="navbar-toggler-icon"></span>
-            </button>
+    <main id="{% block view-id %}{% endblock %}" class="main-container">
+      {% block navbar %}
+        <nav class="navbar navbar-expand-md navbar-dark">
+          {% block navbar-brand %}
+          <a class="navbar-brand" href="/"><img src="{% static 'images/logo-inverse.png' %}" title="Indigo"></a>
+          {% endblock %}
+          {% block navbar-toggle %}
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar-collapse">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          {% endblock %}
```

Comment:
what does this do?

Reply:
the navbar-toggle? it gives you the menu button when on mobile, top right corner.